tests/test_cli.py

```diff
@@ -63,3 +64,12 @@ def check_options_for_duplicates(options: Dict[str, List[str]]) -> Set[str]:
     return set(
         [flag for flag in flags if (flags.count(flag) > 1 or flag in RESERVED_FLAGS)]
     )
+
+
+def test_loaded_modules(runner):
```

Comment:
This test won't work with random test execution order. IMO you have to execute this test in a separate python execution.

The following fails:
```
pytest tests/nativeapp/test_init.py tests/test_cli.py
```

Reply:
Run as separate command in tox